activitystreams-core/index.html

```diff
@@ -3837,6 +3837,220 @@
   </section>
 
   <section>
+    <h2>Example Extension</h2>
+
+    <p>
+        It is possible use multiple vocabularies to cover particular
+        characteristics of the activities like data provenance and annotations,
+        which can compliment the Activity Vocabulary. For example: Eric writes a
+        short note to be shared with his followers. After posting the note, he
+        notices a spelling error. He edits the note and re-posts it. Later, Eric
+        decides that the information in the note is incorrect. He deletes the
+        note.
+    </p>
+
+     <figure><figcaption>A series of activities; creating, editing, and deleting a note.</figcaption>
+<div class="nanotabs">
+  <ul>
+    <li><a href="#ex27-jsonld" class="selected">JSON-LD</a></li>
+    <li><a href="#ex27-rdfa" class="selected">RDFa</a></li>
+    <li><a href="#ex27-turtle" class="selected">Turtle</a></li>
+  </ul>
+  <div id="ex27-jsonld" style="display: block;">
+  <pre class="example highlight json">{
+  "@context": [
+    "http://www.w3.org/ns/activitystreams#",
+    {
+      "oa": "http://www.w3.org/ns/oa#",
+      "prov": "http://www.w3.org/ns/prov#"
+    }
+  ],
+  "@type": "Collection",
+  "items": [
+    {
+      "@id": "http://example.org/activity/20150101000000",
+      "@type": [ "Create", "prov:Activity" ],
+      "actor": {
+        "@id": "http://example.org/#eric",
+        "displayName": "Eric"
+      },
+      "displayName": "Eric wrote a note.",
+      "object": {
+        "@id": "http://example.org/entry/20150101000000",
+        "@type": [ "Note", "prov:Entity" ],
+        "attributedTo": "http://example.org/#eric",
+        "content": "Remember... all I'm offering is the trooth. Nothing more."
+      },
+      "published": "2015-01-01T00:00:00Z"
+    },
+    {
+      "@id": "http://example.org/activity/20150101000059",
+      "@type": [ "Update", "prov:Activity", "oa:Annotate" ],
+      "displayName": "Eric edited a note.",
+      "oa:created": {
```

Comment:
Prefer that this use the simple string form with `oa:created` defined in the `@context` as : 
```json
{
  "oa": "http://www.w3.org/ns/oa#",
  "prov": "http://www.w3.org/ns/prov#",
  "oa:created": {
    "@type": "xsd:dateTime"
  }
}
```
Also, it's `xsd:dateTime`, not `xsd:dateTimeStamp`